backends/python-fastapi/app/routers/staffs.py

```
@@ -12,6 +13,36 @@
 
+_DEFAULT_PAGE_COUNT = 5
+
+
+def _build_pager(count: int, limit: int, offset: int, record_count: int) -> dict:
+    if limit <= 0:
+        limit = 20
+    page_count = max(1, math.ceil(count / limit))
+    last_page_offset = (page_count * limit) - limit
+    if count > 0 and offset > last_page_offset:
+        offset = last_page_offset
+    page = int(math.floor(math.ceil(offset / limit))) + 1 if limit > 0 else 1
```

Ruff (0.15.15)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)